### DISCUSSION TITLE
Is it possible to rename types?

### DISCUSSION THREAD

**Discussion by belak** [rust-lang/rust-bindgen#2514] [Q&A] [answered]
I'm working on an experimental legacy SDL crate and I noticed that a few enums got created with a different style as all the other enums. Is it possible to manually rename those?

There were a bunch of constants (starting with `SDLK_`) which got called SDLKey, even though to match with the rest of the library (and most of the other generated enums), it would be better as `SDL_Key`.

**Comment by pvdrz** [comment, marked as answer]
You can use [`ParseCallbacks::item_name`](https://docs.rs/bindgen/latest/bindgen/callbacks/trait.ParseCallbacks.html#method.item_name) for that.

**Reply by belak** [reply]
That worked like a charm! Thanks!
